### DISCUSSION TITLE
Menu Ordering Issue

### DISCUSSION THREAD

**Discussion by chowadfrepo** [mirumee/saleor-dashboard#3281] [General]
Dear Team,

Navigation Menu Re-Ordering is not enabled or not working in the 3.11.3 and 3.12-dev versions. Here is the [link ](https://situla.bitbit.net/filebin/269f7ebe4b74681cca4e7746e620c7b716f539937d21c3ac7ec834ca013f6704/310b73c002a45d5b0af3127a098d995cd489a716395972586f4a59c814400ae2?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=HZXB1J7T0UN34UN512IW%2F20230303%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230303T112548Z&X-Amz-Expires=30&X-Amz-SignedHeaders=host&response-cache-control=max-age%3D30&response-content-disposition=filename%3D%22Video_2023_03_03-1.webm%22&response-content-type=video%2Fwebm&X-Amz-Signature=ab3bb634c1e84bbab943be1b18dbeecbf62b29d03ecb4e0a219ce331e57b0230)for reference

**Comment by karolkielecki** [comment]
Hi @chowadfrepo,
Could you please provide more details and some screenshots so we can better understand the issue?

**Reply by chowadfrepo** [reply]
Hi @karolkielecki ,
I want to bring the sub-menu under HMC1 and reorder the menu. The following GIFs have a sample activity.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/103019359/222730740-8a4dd69d-b720-41e6-b809-c8b0cd37d8f5.gif)

**Reply by karolkielecki** [reply]
Ok, I've just checked it -- and it's working fine for me with dashboard v3.11.3 + core v3.11.12. Do you receive any errors or something? Could you please check your console?
